lib/fluent/plugin/out_forward.rb

```diff
@@ -382,7 +382,12 @@ def create_transfer_socket(host, port, hostname, &block)
           cert_logical_store_name: @tls_cert_logical_store_name,
           cert_use_enterprise_store: @tls_cert_use_enterprise_store,
 
-          linger_timeout: @send_timeout,
```

Comment:
Does [L399](https://github.com/fluent/fluentd/pull/2959/files#diff-40c0e8c88efdc9a297a9fc63bc2bc69fR399) need the same change?

Reply:
I think L399 needn't to apply same change.